boa/src/syntax/ast/node.rs

```diff
@@ -469,22 +469,7 @@ pub enum Node {
     /// [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/throw
     Throw(Box<Node>),
 
-    /// The `typeof` operator returns a string indicating the type of the unevaluated operand.
-    ///
-    /// Syntax: `typeof operand`
-    ///
-    /// Returns a string indicating the type of the unevaluated operand.
-    ///
-    /// More information:
-    ///  - [ECMAScript reference][spec]
-    ///  - [MDN documentation][mdn]
-    ///
-    /// [spec]: https://tc39.es/ecma262/#sec-typeof-operator
-    /// [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof
-    TypeOf(Box<Node>),
-
-    /// The `try...catch` statement marks a block of statements to try and specifies a response
-    /// should an exception be thrown.
+    /// The `try...catch` statement marks a block of statements to try and specifies a response should an exception be thrown.
```

Comment:
Hmmm why was this moved to one line? Rust recommends having 100 characters per line, was it too short?

Reply:
My IDE must have done this, I'll revert it